src/puppet-web/bridge.ts

```diff
@@ -397,15 +397,14 @@ export class Bridge {
     }
   }
 
-  public sendMedia(toUserName: string, mediaId: string, type: number): Promise<boolean> {
-    if (!toUserName) {
+  public sendMedia(mediaData: any): Promise<boolean> {
```

Comment:
Using a type like `any` can make you feel very convenience at start, But it will cause problems in the future because it disables all the benefits from the static type checking.

So please do not use an `any` type here. I have two suggestions:

1. Keep the original style.

Then we will have a function like:

```ts
public sendMedia(
  toUserName: string,
  mediaId: string,
  msgType,
  fileName,
  fileMd5,
  fileSize,
  fileType,
  mmFileExt,
  mmFileId,
)
```

It's very verbose, but it's more clear and prevent us to make mistakes in the future.

2. Or do it the better way: create an interface for this data type.

```
interface MediaData {
  toUserName: string,
  mediaId: string,
  msgType,
  fileName,
  fileMd5,
  fileSize,
  fileType,
  mmFileExt,
  mmFileId,
}

public sendMedia(mediaData: MediaData): Promise<boolean> {
```

I'll prefer the method 2, but you can use either of them, but not the `any` one.